pyproject.toml

```diff
@@ -4,3 +4,30 @@ requires = [
   "wheel",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.ruff]
+target-version = "py310"
+src = [
+    "src",
+    "tests",
+]
+format.docstring-code-format = true
+lint.select = ['E4', 'E7', 'E9', 'F', 'I']
```

Comment:
Those linter rules were copied from `etk`, where we have already done a good amount of work of selecting good rules that work for us.